<launch>
  <arg name="gripper_type" default="parallel"/>
  <group ns="/cobotta">
    <!-- gripper type: none/parallel/vacuum --> 

    <rosparam command="load" file="$(find denso_cobotta_gripper)/config/gripper_$(arg gripper_type).yaml" />
 
    <!-- node -->
    <node name="denso_cobotta_gripper" pkg="denso_cobotta_gripper" type="denso_cobotta_gripper" output="screen" required="true">
      <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}]: ${node}: ${message}"/>
    </node>
  </group>
</launch>
